src/lando/tests/test_version.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+import subprocess
+
+
+def test_version():
+    # explicitly generate the version.py file since we can't
+    # guarantee it already exists in the testing environment
+    subprocess.call(["lando", "generate_version_file"])
```

Comment:
If you define the functionality that generates and stores the file in a method in `lando.utils.management.commands.generate_version_file`, then you can simply import the method and call it here instead.